projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -99,6 +99,8 @@ public function get_item( $request ) { // phpcs:ignore Generic.CodeAnalysis.Unus
 	public function prepare_menu_for_response( array $menu ) {
 		global $submenu;
 
+		do_action( 'admin_menu_rest_api' );
+		global $menu;
```

Comment:
I'm not sure I understand why we need this here, since we already get `$menu` from the function parameter. Could you clarify things a bit?